lib/src/components/NxLoadingSpinner/__tests__/NxLoadingSpinner.test.tsx

```diff
@@ -5,22 +5,26 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { shallow } from 'enzyme';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 import NxLoadingSpinner from '../NxLoadingSpinner';
 
 describe('NxLoadingSpinner', function() {
+  const quickRender = rtlRender(NxLoadingSpinner, {}),
+      renderEl = rtlRenderElement(NxLoadingSpinner, {});
+
   it('renders an nx-loading-spinner__icon within a nx-loading-spinner', function() {
-    expect(shallow(<NxLoadingSpinner />)).toHaveClassName('.nx-loading-spinner');
-    expect(shallow(<NxLoadingSpinner />)).toContainMatchingElement('.nx-loading-spinner__icon');
+    const { container } = quickRender(<NxLoadingSpinner />);
+    expect(container.querySelector('.nx-loading-spinner__icon')).toBeInTheDocument();
   });
 
   it('renders the specified children', function() {
```

Comment:
Not part of the publicly documented API; doesn't need to be tested

Reply:
> Not part of the publicly documented API; doesn't need to be tested

Removed it here: b2e3ca44151695a000534f912ba8fc0217317bb4